### ISSUE TITLE
training will randomly freeze for training Unable to train

### ISSUE THREAD

**Issue by spencersweet0530** [lukemelas/EfficientNet-PyTorch#169]
main.py:100: UserWarning: You have chosen a specific GPU. This will completely disable data parallelism.
  warnings.warn('You have chosen a specific GPU. This will completely '
Use GPU: 0 for training
Loaded pretrained weights for efficientnet-b3
=> using pre-trained model 'efficientnet-b3'
Using image size 300
Test: [0/4]	Time  1.406 ( 1.406)	Loss 8.2919e+00 (8.2919e+00)	Acc@1   0.00 (  0.00)	Acc@5   0.00 (  0.00)
 * Acc@1 0.000 Acc@5 0.000

Is there a workaround?
is stop there and stop training
